app/uk/gov/hmrc/fhregistrationfrontend/views/emailverification/email_options.scala.html

```diff
@@ -51,8 +51,9 @@
   <link rel="stylesheet" href='@routes.Assets.at("stylesheets/fh.css")'/>
 
   @Back(navigation)
+    <a href="@uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.businessType()" class="link-back">@Messages("fh.generic.back")</a>
```

Comment:
can we check for an already registered user if they do an amendment if the back for this page is the same